service/src/main/java/bio/terra/profile/service/crl/AzureCrlService.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.profile.service.crl;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.profile.app.configuration.AzureConfiguration;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.resourcemanager.costmanagement.CostManagementManager;
+import com.azure.resourcemanager.managedapplications.ApplicationManager;
+import com.azure.resourcemanager.resources.ResourceManager;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class AzureCrlService {
+
+  private final AzureConfiguration azureConfiguration;
+  private final ClientConfig clientConfig;
+
+  @Autowired
+  public AzureCrlService(AzureConfiguration azureConfiguration, ClientConfig clientConfig) {
+    this.azureConfiguration = azureConfiguration;
+    this.clientConfig = clientConfig;
+  }
+
+  public ResourceManager getResourceManager(UUID subscriptionId) {
+    return getResourceManager(
+        UUID.fromString(azureConfiguration.managedAppTenantId()), subscriptionId);
+  }
+
+  /** Returns an Azure {@link ResourceManager} configured for use with CRL. */
+  public ResourceManager getResourceManager(UUID tenantId, UUID subscriptionId) {
+    AzureProfile azureProfile =
+        new AzureProfile(tenantId.toString(), subscriptionId.toString(), AzureEnvironment.AZURE);
```

Comment:
I assume this is existing code, but it's odd that the you have to specify `AzureEnvironment.AZURE` when creating an `AzureProfile`.

Reply:
Yeah, I thought so too.
Looking at `AzureEnvironment`, it appears there are a bunch of alternatives specific azure instances, like `AZURE_US_GOVERNMENT`, `AZURE_GERMANY`, and `AZURE_CHINA`.